tests/foreman/api/test_repository.py

```diff
@@ -1387,6 +1387,46 @@ def test_positive_sync_rh_app_stream(self):
         """
         pass
 
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_bulk_cancel_sync(self):
+        """Bulk cancel 10+ repository syncs
+
+        :id: f9bb1c95-d60f-4c93-b32e-09d58ebce80e
+
+        :steps:
+            1. Add 10+ repos and sync all of them
+            2. Cancel all of the syncs
+            3. Check /var/log/foreman/production.log and /var/log/messages
+
+        :expectedresults: All the syncs stop successfully.
+
+        :CaseImportance: High
+
+        :CaseAutomation: NotAutomated
+        """
+        pass
+
+    @pytest.mark.stubbed
+    @pytest.mark.tier1
+    def test_positive_reboot_recover_sync(self):
```

Comment:
These tests doesn't seem like `tier1`, maybe they should be `tier3` or `tier4`.
Also, they seem like destructive tests so we should also add 
```
@pytest.mark.destructive
@pytest.mark.run_in_one_thread
```

Reply:
Good suggestion. I'd recommend you reconsider the tiering proposal for most of these

Reply:
I changed most of them to `tier3` except for the one that just cancels repos syncing. However, I felt like `test_positive_reboot_recover_sync` was possibly `tier4` as a reboot and sync could take a long time. If this doesn't fit the criteria for `tier4` I can update it to `tier3`.